dev-infrastructure/docs/development-setup.md

```diff
@@ -353,8 +355,44 @@ Then register it with the Maestro Server
     ```
     az network vnet subnet update -g <resource-group> -n <subnet-name> --vnet-name <vnet-name> --network-security-group <nsg-name>
     ```
+  - Generate a random alphanumeric string used as a suffix for the User-Assigned Managed Identities of the operators of the cluster
```

Comment:
Although for now we document this, I think the whole step 2, which is creation of prerequisites could be automated or partially automated by having a script that executes the commands.

We can work on that in the near future if we want.

Reply:
+1 that would be great.